data/human/weapons.txt

```diff
@@ -33,6 +33,36 @@ outfit "Energy Blaster"
 		"hull damage" 6.6
 	description "Although not the most accurate or damaging of weapons, the Energy Blaster is popular because it is small enough to be installed on even the tiniest of ships. One blaster is not enough to do appreciable damage to anything larger than a fighter, but a ship equipped with several of them becomes a force to be reckoned with."
 
+outfit "Twin Blaster"
+	category "Guns"
+	cost 40000
+	thumbnail "outfit/twin blaster"
+	"mass" 12
+	"outfit space" -12
+	"weapon capacity" -12
+	"gun ports" -1
+	weapon
+		sound "blaster"
+		"submunition" "Blaster Submunition"
+			"offset" 1.5 0
+		"submunition" "Blaster Submunition"
+			"offset" -1.5 0
```

Comment:
New proposal from @Hecter94: have the offsets be close at the barrel, but give each projectile a facing able so that they spread out over range.
```suggestion
			"offset" 1 0
			"facing" 0.3
		"submunition" "Blaster Submunition"
			"offset" -1 0
			"facing" -0.3
```